src/codegen/doc/mod.rs

```diff
@@ -555,6 +558,62 @@ fn create_object_doc(w: &mut dyn Write, env: &Env, info: &analysis::object::Info
             configured_signals.iter().all(|s| s.generate_doc),
         )?;
     }
+
+    for function in virtual_methods {
+        let configured_virtual_methods = obj.virtual_methods.matched(&function.name);
+        let is_manual = configured_virtual_methods.iter().any(|f| f.status.manual());
+        let (ty, object_location) = if let Some(trait_name) = configured_virtual_methods
+            .iter()
+            .find_map(|f| f.doc_trait_name.as_ref())
+        {
+            (
+                TypeStruct::new(SType::Trait, trait_name),
+                // Because we cannot sensibly deduce where the docs end up,
+                // assume they're outside the docs so that no `Self::` links
+                // are generated.  It is currently quite uncommon to specify
+                // the `{}Manual` trait, which would be ObjectLocation::ExtManual.
+                None,
+            )
+        } else if is_manual {
+            (
+                TypeStruct::new(SType::Trait, &format!("{}ImplManual", info.name)),
```

Comment:
What's this? :)

Reply:
This is unused so far, maybe I should drop it until we get code generations

Reply:
Yeah. This is exactly the case that must not happen when we generate these things.